### PR TITLE
IDE-771 Insert file grayed out when right click on added ECL folder

### PR DESCRIPTION
**Pull request by dehilsterlexis**
Fixed IDE-771

Signed-off-by: dehilsterlexis <david.dehilster@lexisnexis.com>

### CODE REVIEW

**Comment by dehilsterlexis** [comment]
@GordonSmith for reivew

File: clib/cmdProcess.h [hunk 1]
```diff
@@ -3,4 +3,4 @@
 
 int CLIB_API runProcess(const std::_tstring &command, const std::_tstring &directory, const std::_tstring &path, const std::_tstring &in, std::_tstring &out, std::_tstring &err);
 
-
+#define PLUGINS_FOLDER _T("plugins")
```
Comment (GordonSmith) [inline clib/cmdProcess.h]: Move "#define"s to the top of the file for consistency

File: comms/DiskModule.cpp [hunk 2]
```diff
@@ -416,7 +417,7 @@ class CDiskModule : public IDiskModule, public clib::CLockableUnknown
     bool IsPlugin() const
     {
         clib::recursive_mutex::scoped_lock proc(m_mutex);
-        return m_plugin || m_parent == NULL;
```
Comment (GordonSmith) [inline comms/DiskModule.cpp]: I think (from memory) that "IsPlugin" might also be used for coloring the root folders and for some special handling for "root" folders.  Can you track down all the places it is currently being called from and double check?

If so, we may want two methods:  "IsPlugin" and "IsRoot"

**Comment by dehilsterlexis** [comment]
@GordonSmith added new folder type "Top Folder" with a greenish color. Back to you.

**Pull request merged**